apiserver/facades/client/modelmanager/modelmanager.go

```diff
@@ -1001,11 +1002,14 @@ func (m *ModelManagerAPI) getModelInfo(tag names.ModelTag) (params.ModelInfo, er
 		return params.ModelInfo{}, errors.Trace(err)
 	}
 
+	isControllerModel := st.ControllerModelUUID() == model.UUID()
```

Comment:
You should just use `st.IsController()`

Reply:
I've needed to add an `IsController` method to model as that `st` object isn't actually an instance of `state.State`